target/scripts/start-mailserver.sh

```diff
@@ -129,8 +129,8 @@ function _register_functions
   [[ ${SMTP_ONLY}               -ne 1 ]] && _register_start_daemon '_start_daemon_dovecot'
 
   [[ ${ENABLE_UPDATE_CHECK}     -eq 1 ]] && _register_start_daemon '_start_daemon_update_check'
-  [[ ${ENABLE_RSPAMD}           -eq 1 ]] && _register_start_daemon '_start_daemon_rspamd'
   [[ ${ENABLE_RSPAMD_REDIS}     -eq 1 ]] && _register_start_daemon '_start_daemon_rspamd_redis'
+  [[ ${ENABLE_RSPAMD}           -eq 1 ]] && _register_start_daemon '_start_daemon_rspamd'
```

Comment:
Maybe add a comment, that RSPAM startup order matters?

Reply:
Done in [1ecdfe6](https://github.com/docker-mailserver/docker-mailserver/pull/3206/commits/1ecdfe60abc48ec11ef1238455aa7bac686dec14)